crates/archive/src/lib.rs

```diff
@@ -18,7 +18,7 @@ pub enum ArchiveError {
     HttpError(attohttpc::StatusCode),
 
     #[error("HTTP header '{0}' not found")]
-    MissingHeaderError(String),
+    MissingHeaderError(attohttpc::header::HeaderName),
```

Comment:
It looks like the [`Header::name`](https://docs.rs/headers/latest/headers/trait.Header.html#tymethod.name) associated function returns a `&'static` reference to the `HeaderName` object. Given that, we can probably use the same here (`&'static HeaderName`) instead of needing to convert it to an owned value.

Reply:
Updated to use `&'static HeaderName`.